escoot/de-at/augment-gemeinsamer-service-und-mietvertrag-de-at.md

```diff
@@ -28,7 +28,7 @@ Auslieferung eines E-Scooters an den Kunden zu Vertragsbeginn
 
 Das Anbringen von normalem **E-Scooter**-Zubehör, wie z.B. einer Handyhalterung, am **E-Scooter** ist erlaubt, sofern es sich spurlos wieder entfernen lässt. Das Anbringen und Entfernen von Zubehör sowie die Reparatur von Spuren, die das Zubehör hinterlässt, gehen auf Kosten des **Kunden**.
 
-<div class="page"></div>
+<div class="page"/>
```

Comment:
Page breaks should follow [special markup](https://github.com/augment-eco/terms-and-conditions?tab=readme-ov-file#special-markup).

```suggestion
<div class="page"></div>
```